x-pack/legacy/plugins/siem/scripts/loop_cypress_tests.js

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+const fs = require('fs');
+const os = require('os');
+const process = require('process');
+const spawn = require('child_process').spawn;
+/* eslint-disable no-process-exit */
+const MUST_RUN_FROM_DIR = 'kibana';
+const OUTPUT_FILE = `target/loop-cypress-tests.txt`;
```

Comment:
If the `target` directory does not exist, the script fails with the following error:

```
➜  kibana git:(cypress-jenkins-preparation) node x-pack/legacy/plugins/siem/scripts/loop_cypress_tests.js 1

Cypress tests will be run 1 times

Test output will be appended to 'target/loop-cypress-tests.txt'


*** Starting test run 1 of 1 on host tizelastic.local at Mon Mar 02 2020 11:31:59 GMT-0700 (Mountain Standard Time) ***


(node:55574) UnhandledPromiseRejectionWarning: Error: ENOENT: no such file or directory, open 'target/loop-cypress-tests.txt'
    at Object.openSync (fs.js:443:3)
    at Object.writeFileSync (fs.js:1194:35)
    at Object.appendFileSync (fs.js:1240:6)
    at runNTimes (/Users/abgoldst/Projects/forks/MadameSheema/kibana/x-pack/legacy/plugins/siem/scripts/loop_cypress_tests.js:62:8)
```

Consider changing:

```
const OUTPUT_FILE = `target/loop-cypress-tests.txt`;
```

to:

```
const OUTPUT_DIR = 'target';
const OUTPUT_FILE = `${OUTPUT_DIR}/loop-cypress-tests.txt`;
const createOutputDir = () => {
  fs.mkdir(OUTPUT_DIR, { recursive: true }, err => {
    if (err) throw err;
  });
};
```

...and invoking the new `createOutputDir` function just before invoking `runNTimes(timesToRun)`, i.e. the following:

```
createOutputDir();
runNTimes(timesToRun);
```

Reply:
Done :)